packages/core/types/src/http/fulfillment/admin/payloads.ts

```diff
@@ -0,0 +1,46 @@
+interface AdminCreateFulfillmentItem {
+  title: string
+  sku: string
+  quantity: number
+  barcode: string
+  line_item_id?: string
+  inventory_item_id?: string
+}
+
+interface AdminCreateFulfillmentLabel {
+  tracking_number: string
+  tracking_url: string
+  label_url: string
+}
+
+interface AdminFulfillmentDeliveryAddress {
+  first_name?: string
+  last_name?: string
+  phone?: string
+  company?: string
+  address_1?: string
+  address_2?: string
+  city?: string
+  country_code?: string
+  province?: string
+  postal_code?: string
+  metadata?: Record<string, string> | null
+}
+
+export interface AdminCreateFulfillment {
+  location_id: string
+  provider_id: string
+  delivery_address: AdminFulfillmentDeliveryAddress
+  items: AdminCreateFulfillmentItem[]
+  labels: AdminCreateFulfillmentLabel[]
+  // Validator requires an empty object for the order field
+  // seems like an issue with the validator but including it here
+  // to make the type match the validator.
+  order: Record<string, unknown>
```

Comment:
The validator for this payload seems to be off, but I have matched it to prevent any errors.

Reply:
Yeah I also had some issues now that I was moving some tests, this is fine for now, just add a TODO please so it's a bit easier to find it.